src/plugins/intel_gna/src/layers/gna_split_layer.hpp

```diff
@@ -73,22 +72,21 @@ inline std::pair<int64_t, std::vector<uint32_t>> AlignedSplitSizesPerAxis(Infere
     IE_ASSERT(firstValuableDim != std::end(dims));
     auto splittedElementsSize = *firstValuableDim;
     auto splittedDimIx = std::distance(std::begin(dims), firstValuableDim);
-    auto alignment = limitations::Limitations::kInputByteAlignment;
+    auto alignment = limitations::Limitations::get_instance()->get_memory_alignment();
```

Comment:
this is not a part of this PR, but would like to point you that it would be better to implement singleton for limitation instance.